schemas/v3.1/schema.yaml

```
@@ -607,6 +607,12 @@ $defs:
       externalValue:
         type: string
         format: uri
+    not:
+      allOf:
+      - required:
+        - value
+      - required:
+        - externalValue
```

Can this be simplified like so?
```suggestion
      required:
        - value
        - externalValue
```

---

yes. good call!